### PR TITLE
housekeeping

### PR DESCRIPTION
**Pull request by alarbada**
### Description

- Made linter config stricter
- Fix the consequent linter errors
- Track used tools in `tools.go`
- Improve github linter workflow
- Fix missing "alarbada -> conduit labs" rename

### Quick checks:

- [x] There is no other [pull request](https://github.com/alarbada/conduit-connector-rabbitmq/pulls) for the same update/change.
- [x] I have written unit tests.
- [x] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

**Review by raulb — DISMISSED** [review]

File: paramgen_src.go [hunk 1]
```diff
@@ -9,31 +9,91 @@ import (
...
 			Validations: []sdk.Validation{},
 		},
-		"queueName": {
-			Default:     "",
-			Description: "queueName is the name of the queue to consume from / publish to",
-			Type:        sdk.ParameterTypeString,
-			Validations: []sdk.Validation{
-				sdk.ValidationRequired{},
-			},
+		"tls.enabled": {
```
Comment (lovromazgon) [inline paramgen_src.go:92]: This config parameter is missing in the readme.

File: paramgen_dest.go [hunk 2]
```diff
@@ -9,24 +9,6 @@ import (
...
-			Type:        sdk.ParameterTypeString,
-			Validations: []sdk.Validation{},
-		},
-		"clientKey": {
-			Default:     "",
-			Description: "clientKey is the path to the client key to use for TLS",
-			Type:        sdk.ParameterTypeString,
-			Validations: []sdk.Validation{},
-		},
 		"delivery.appID": {
```
Comment (lovromazgon) [inline paramgen_dest.go:12]: `delivery.contentType` is also missing in the readme (well, it's named `contentType`).

File: paramgen_dest.go [hunk 3]
```diff
@@ -153,26 +138,50 @@ func (DestinationConfig) Parameters() map[string]sdk.Parameter {
 			Type:        sdk.ParameterTypeBool,
 			Validations: []sdk.Validation{},
 		},
+		"queue.name": {
```
Comment (lovromazgon) [inline paramgen_dest.go:141]: Btw, why don't we put the `queue.*` configuration options in the shared `Config`? These config values seem to be present both in the source and destination.

**Review by lovromazgon — APPROVED** [review]
Looks good 💪

**Pull request merged**